l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -869,7 +869,7 @@ def invoiceCreate(
             'currency_id': currency[0].id,
             'journal_id': purchase_journal.id,
             # 'origin': xmlData.datiOrdineAcquisto,
-            'fiscal_position': False,
+            'fiscal_position': partner.property_account_position.id,
```

Comment:
Ciao @SimoRubi, 
che io sappia in v8 quando una m2o non non è impostata non viene restituito recordset vuoto ma False, da qui la necessità di fare il controllo o aggiungere " or False".

Reply:
Ciao, grazie della review!
Ho fatto un po' di debug in locale e mi sembra che il comportamento sia in linea con quello di v10 quindi viene restituito un record vuoto, credo che il record vuoto venga restituito in https://github.com/odoo/odoo/blob/4ca06e02558b202183592f4d7c67f76f302bc0d1/openerp/fields.py#L1577

Inoltre, se fosse come dici i test fallirebbero perché avremmo `partner.property_account_position = False` (i clienti generati nei test non hanno posizione fiscale) e quindi verrebbe restituito il classico errore `AttributeError: 'bool' object has no attribute 'id'` quando viene eseguito `partner.property_account_position.id`

Reply:
Infatti pensavo proprio a quell'errore, ma se hai fatto il test allora ok!

Reply:
@vincenzoterzulli ho fatto il test, ma come dicevo non ti devi per forza fidare del mio test: se fosse come dici in https://github.com/OCA/l10n-italy/pull/1456#discussion_r363792644 fallirebbero i test automatici